connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -188,6 +188,7 @@ class PlanGenerationTestSuite
     .add("e", "array<int>")
     .add("f", MapType(StringType, simpleSchema))
     .add("g", "string")
+    .add("h", "boolean")
```

Comment:
let 's do not add field here, it cause too many changes.

you can add a new one if needed